src/OpenTelemetry.Instrumentation.EntityFrameworkCore/OpenTelemetry.Instrumentation.EntityFrameworkCore.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry.Api" Version="1.2.0" />
+    <PackageReference Include="OpenTelemetry.Api" Version="$(OpenTelemetryCoreLatestVersion)" />
```

Comment:
lets update this to 1.3.2 instead?
https://www.nuget.org/packages/OpenTelemetry/1.3.2

Reply:
Agree. Lets only do a release after 1.3.2 change is also done.